lib/src/license_detection/tokenizer.dart

```diff
@@ -0,0 +1,164 @@
+import 'dart:collection';
+import 'dart:convert';
+
+import 'package:meta/meta.dart';
+// import 'package:source_span/source_span.dart';
+
+@sealed
+class Token {
+  final String token;
+  final int? position;
+  final int line;
+
+  // TODO : Handle offset and location using source span.
+  // final SourceSpan sourceSpan;
+
+  Token(this.token, this.position, this.line);
+}
+
+const lineSplitter = LineSplitter();
+
+/// Create tokens and
+/// normalizes them using the approach of [google licenseClassifier]
+/// to provide better chances of matching.
+/// [google licenseClassifier]  https://github.com/google/licenseclassifier/blob/bb04aff29e72e636ba260ec61150c6e15f111d7e/v2/tokenizer.go#L85
```

Comment:
Make it correct markdown:
```suggestion
/// [google licenseClassifier]: https://github.com/google/licenseclassifier/blob/bb04aff29e72e636ba260ec61150c6e15f111d7e/v2/tokenizer.go#L85
```